examples/vue/valibot/package.json

```diff
@@ -9,34 +9,15 @@
     "serve": "vite preview"
   },
   "dependencies": {
-    "@tanstack/form-core": "0.11.0",
     "@tanstack/vue-form": "0.11.0",
     "@tanstack/valibot-form-adapter": "0.11.0",
     "vue": "^3.3.4",
-    "valibot": "^0.20.1",
-    "@tanstack/zod-form-adapter": "0.11.0",
-    "@tanstack/react-form": "0.11.0",
-    "@tanstack/yup-form-adapter": "0.11.0",
-    "@tanstack/solid-form": "0.11.0"
```

Comment:
This is theoretically nice to remove these packages, but they are added back automatically on every release via a CI/CD commit. We'll need to address this root cause first before merging

Reply:
How strange! I'll have a look soon.

Reply:
Fixed! It was the `Updating all example dependencies...` block in `scripts/publish.ts`. Shouldn't be necessary since pnpm will resolve to the internal workspace and users will resolve the latest semantic version. We don't have this in tanstack-query anymore.